### PR TITLE
Add katex header & bump libs version

### PR DESCRIPTION
**Pull request by survived**
Fixes the build failure on docs.rs. Releases a new version of the lib `v0.1.1`

### CODE REVIEW

File: generic-ec-core/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "generic-ec-core"
-version = "0.1.0"
+version = "0.1.2"
```
Comment (maurges) [inline generic-ec-core/Cargo.toml:3]: Why are this and generic-ec bumped to 1.2?

Comment (survived) [reply generic-ec-core/Cargo.toml:3]: because I managed to fix the issue only after second attempt

Comment (survived) [reply generic-ec-core/Cargo.toml:3]: you can see the git history

Comment (maurges) [reply generic-ec-core/Cargo.toml:3]: I see, it's because the versions were published to crates.io. I would vote for bumping all versions to 1.2 then

Comment (survived) [reply generic-ec-core/Cargo.toml:3]: Why bumping what has not been changed? If they were supposed to have the same version number, they would be a single lib

Comment (maurges) [reply generic-ec-core/Cargo.toml:3]: But nothing changed in all 4 crates, according to semver. We could even yank the old version and republish with the same number.

I thought they are different crates to ease development and namespacing, but they are still interconnected. Sitting here thinking about it, I can see weak technical arguments for simultaneous and not bumping of versions, and my strongest feeling here is aesthetics which also doesn't matter much. So I leave it to your choice.

**Review by maurges — APPROVED** [review]

File: generic-ec-core/Cargo.toml (hunk 1 above)
Comment (survived) [reply generic-ec-core/Cargo.toml:3]: If I could reupload library with the same version I'd do this, but it's impossible. Yanking doesn't delete crate version, it just marks it "do not use me", which is not even mandatory.

Comment (survived) [reply generic-ec-core/Cargo.toml:3]: I don't see strong arguments in favor of bumping all 4 crates, so I leave them as they are

**Pull request merged**